pkg/persistence/dgraph/pod.go

```diff
@@ -97,26 +108,30 @@ func PersistPod(pod api_v1.Pod) error {
 	return err
 }
 
-func GetContainers(pod api_v1.Pod, podUid string) []*Container {
+// GetContainers fetchs the list of containers in given pod
+func GetContainers(pod api_v1.Pod, podUID string) []*Container {
 	podXid := pod.Namespace + ":" + pod.Name
 
 	containers := []*Container{}
 	for _, c := range pod.Spec.Containers {
 		containerXid := podXid + ":" + c.Name
-		uid, _ := GetUId(Client, containerXid, IsContainer)
+		uid, err := GetUID(Client, containerXid, IsContainer)
+		if err != nil {
```

Comment:
no need for err check, GetUID returns uid as "" if err is not nil

Reply:
Same as above.